packages/hooks/tests/useCurrentUser.test.js

```diff
@@ -12,27 +11,27 @@ afterEach(() => {
 });
 
 const Component = () => {
-  const { data: user, status, error } = useCurrentUser();
+  const { data, isFetching, error } = useCurrentUser({
+    cacheTime: 0,
+    retry: false,
+  });
+
+  if (isFetching) return <span data-testid="loading" />;
+  if (data) return <span data-testid="valid">{JSON.stringify(data)}</span>;
+  if (error) return <span data-testid="invalid">An error occurred</span>;
 
-  return (
-    <ReactQueryConfigProvider config={{ cacheTime: 0, retry: false }}>
-      {status === 'loading' ? (
-        <span data-testid="loading" />
-      ) : (
-        JSON.stringify(error || user)
-      )}
-    </ReactQueryConfigProvider>
-  );
+  return null;
 };
 
 describe('useCurrentUser', () => {
   test('should set error on rejected promise', async () => {
     avUserApi.me.mockRejectedValueOnce('An error occurred');
 
-    const { getByText } = render(<Component />);
+    const { getByTestId } = render(<Component />);
 
-    await waitForElement(() => getByText('"An error occurred"'));
+    await waitForElement(() => getByTestId('invalid'));
```

Comment:
This will fail the test if the element isn't found, but I think it's still a best practice to have an assertion here that the element exists/is not undefined